static/app/components/profiling/functionsMiniGrid.tsx

```diff
@@ -1,4 +1,4 @@
-import {CSSProperties, Fragment} from 'react';
+import React, {CSSProperties, Fragment} from 'react';
```

Comment:
We try not to use top level `React` imports and import just what we need.

Reply:
✅